web/src/components/tasks/RaffleBingoPaginator.tsx

```diff
@@ -0,0 +1,193 @@
+import { Content } from "@/components/Content";
+import { HorizontalLine } from "@/components/HorizontalLine";
+import { CtaContainer } from "@/components/njwds-extended/cta/CtaContainer";
+import { SingleCtaLink } from "@/components/njwds-extended/cta/SingleCtaLink";
+import { HorizontalStepper } from "@/components/njwds-extended/HorizontalStepper";
+import { PrimaryButton } from "@/components/njwds-extended/PrimaryButton";
+import { SecondaryButton } from "@/components/njwds-extended/SecondaryButton";
+import { ActionBarLayout } from "@/components/njwds-layout/ActionBarLayout";
+import { TaskHeader } from "@/components/TaskHeader";
+import { TaskStatusChangeSnackbar } from "@/components/TaskStatusChangeSnackbar";
+import { fetchTaskByFilename } from "@/lib/async-content-fetchers/fetchTaskByFilename";
+import { useConfig } from "@/lib/data-hooks/useConfig";
+import { useUserData } from "@/lib/data-hooks/useUserData";
+import { StepperStep, Task } from "@/lib/types/types";
+import { LookupTaskAgencyById } from "@businessnjgovnavigator/shared";
+import { TaskProgress } from "@businessnjgovnavigator/shared/userData";
+import { ReactElement, ReactNode, useEffect, useState } from "react";
+
+interface Props {
+  task: Task;
+  CMS_ONLY_STEPINDEX?: number;
+}
+
+export const RaffleBingoPaginator = (props: Props): ReactElement => {
+  const [stepIndex, setStepIndex] = useState(props.CMS_ONLY_STEPINDEX ?? 0);
+  const [stepLabels, setStepLabels] = useState<StepperStep[]>([]);
```

Comment:
[sand] I don't think you need to be tracking this with state. This isn't dynamic.

```suggestion
  let stepLabels: StepperStep[];
```

In the `useEffect` you could modify `getStepLabels` to just `stepLabels = labels`, or modify the function to return labels and say `stepLabels = getStepLabels()`